src/CollAction/Models/ProjectViewModels/DisplayProjectViewModel.cs

```diff
@@ -15,6 +16,21 @@ public class DisplayProjectViewModel
 
         public int Participants { get; set; }
 
+        public bool HasDescriptionVideo { get { return Project.DescriptionVideoLink != null; } }
+
+        public string DescriptionVideoYouTubeEmbedLink {
+            get {
+                if (!HasDescriptionVideo) { return ""; }
+
+                var regex = new Regex(@"^(?:https?:\/\/www\.youtube\.com\/watch\?v=)((?:\w|-){11,})(?:\S+)?$");
```

Comment:
Prehaps extract this to a validation attribute? Also, might better to use a url parser for stuff like this, a regex isn't the best way to parse urls.